Diagnostic/diagnostic.py

```diff
@@ -396,6 +396,18 @@ def setEventVolume(mdsdCfg,ladCfg):
             
     XmlUtil.setXmlValue(mdsdCfg,"Management","eventVolume",eventVolume)
         
+def getStorageAccountEndPoint(account):
+    endpoint = readPrivateConfig('storageAccountEndPoint')
+    if endpoint:
```

Comment:
there is some indention problem (python is strict for that). The line 407's else is for line 401 but the indention doesn't show that. Also from line 403 to 406, they should indent to right